container/apptabs.go

```diff
@@ -3,678 +3,312 @@ package container
 import (
 	"fyne.io/fyne/v2"
 	"fyne.io/fyne/v2/canvas"
-	"fyne.io/fyne/v2/driver/desktop"
-	"fyne.io/fyne/v2/internal"
 	"fyne.io/fyne/v2/layout"
 	"fyne.io/fyne/v2/theme"
 	"fyne.io/fyne/v2/widget"
 )
 
+// MaxAppTabs is the maximum number of visible tabs in an AppTab.
+// Any addition tabs are moved to the overflow menu.
+const MaxAppTabs = 7
```

Comment:
I'm not sure about this constant, but if it is needed can it be an unexported constant?

Reply:
Done